### PR TITLE
Twig debug

### PR DESCRIPTION
**Pull request by Chi-teck**
This adds Drush command to enable/disable Twig debug mode.

This was previously requested as part of #5109. However, the proposed `site:mode` command is too wide, indeed. And it may need a lot of options to fit everyone's needs.

Unlike most of `site:mode` settings Twig debug mode cannot be set through settings.php or `drush cset`. It requires changing Twig parameters in `development.services.yml` file and rebuilding container. However, since Drupal 10.1.0 there is [a way](https://www.drupal.org/node/3359728) to enable it through Drupal states. So that the `drush twig:debug` command can be really simple.

Relevant issue on drupal.org
https://www.drupal.org/project/drupal/issues/3363676

### CODE REVIEW

File: src/Commands/core/TwigCommands.php [hunk 1]
```diff
@@ -117,4 +122,32 @@ public function twigCompile(): void
             $this->logger()->success(dt('Compiled twig template !path', ['!path' => $relative]));
         }
     }
+
+    /**
+     * Enables Twig debug and disables caching Twig tempaltes.
```
Comment (weitzman) [inline src/Commands/core/TwigCommands.php]:
```suggestion
     * Enables Twig debug and disables caching Twig templates.
```

File: src/Commands/core/TwigCommands.php [hunk 2]
```diff
@@ -117,4 +122,32 @@ public function twigCompile(): void
...
+     *
+     * @see \Drupal\system\Form\DevelopmentSettingsForm::submitForm()
+     */
+    #[CLI\Command(name: self::DEBUG, aliases: ['twig-debug'])]
+    #[CLI\Argument(name: 'mode', description: 'Debug mode (on/off).')]
```
Comment (weitzman) [inline src/Commands/core/TwigCommands.php]:
```suggestion
    #[CLI\Argument(name: 'mode', description: 'Debug mode. Recognized values: <info>on</info>, <info>off</info>.', suggestedValues: ['on', 'off'])]
```

**Review by weitzman — CHANGES_REQUESTED** [review]

**Comment by Chi-teck** [comment]
Fixed version number comparison.

**Comment by gitressa** [comment]
Thanks! I tried enabling this feature, but can't see any changes ... where should I see the effect of this command?

```
$ drush twig:debug on
 [OK] Enabled twig debug.
```

**Comment by Chi-teck** [comment]
That's just a CLI interface for admin/config/development/settings.

Apparently, it's no longer working because of https://www.drupal.org/node/3437162

**Comment by weitzman** [comment]
Ugh. I wish Drupal would make an API for stuff like this. Then callers like Drush would be isolated from implementation shanges. Raw key-value is not an API.

**Pull request merged**